client/src/components/DishPhotos.jsx

```diff
@@ -1,28 +1,78 @@
 import React from 'react';
+import $ from 'jquery';
+import faker from 'faker';
```

Comment:
consider removing faker from the components. It will remove over 1000 modules in your bundle. It helped me out

Reply:
Just saved your computer from running my component later, bundle.js file went from 8.33 MiB with 1105 modules to 3.2 MiB with 57 modules